### ISSUE TITLE
Translations templates on Transifex are out-of-date

### ISSUE THREAD

**Issue by claudep** [Bouke/django-two-factor-auth#353]
I noticed that several strings were not translated in French in some templates in 1.11.0.
I tracked the issue in Transifex which is still using old en templates.

Could you please check that in Transifex and upload the latest en django.po files?

**Comment by moggers87** [comment]
Yup, getting a Transifex account so I can sort out translations is on my todo list

**Comment by claudep** [comment]
Thanks, it's a bit of a blocker right now to upgrade for me.

**Comment by moggers87** [comment]
Just a quick update: I have a Transifex account and now I'm waiting to be given admin rights so I can push the new strings up. Sorry for the delay.

Also: would I be correct in thinking these strings broke because of #329 ? That might be something we can fix outside of Transifex in the meantime.

**Comment by claudep** [comment]
Sure it is related to #329. But if you setup Transifex to get the po file from GitHub (like we do for Django for example), the templates are automatically updated and you can "simply" fetch translations before each release.

**Comment by moggers87** [comment]
I'll look into setting that up once I have access :+1:

**Comment by moggers87** [comment]
@Bouke how easy would it be to do what @claudep suggested and get transifex to grab the po files automatically?

**Comment by Bouke** [comment]
@moggers87 Grabbing the files is the easy part I think: there needs to be an API token for accessing transifex, that can be stored in a "action secret". But what should the process look like? Running on a schedule, auto-committing to the master branch? Or are we going to do releases from Github Actions as well, and fetch translations is part of that workflow?

**Comment by moggers87** [comment]
[These docs](https://docs.transifex.com/transifex-github-integrations/github-tx-ui) appear to detail how to Transifex can be intregrated with a Github repo. I assume this is what Django uses as their repo has not Github actions that I can see.

It looks like that has the option of opening PRs for new translations, which I'd quite like as: a) a quick manual review is always good b) I am likely to forget about pulling in new translations otherwise

**Comment by claudep** [comment]
Don't you have a release checklist anyway? I'm still old-school at this regard, and like a manual (and local) `tx pull` command so I can have a look at what changed globally at translation level. But I can understand others prefer a full automated solution.

**Comment by Bouke** [comment]
Yep, the checklist can be found [in the readme](https://github.com/Bouke/django-two-factor-auth#releasing). A fully automated solution would be helpful as to standardise the output and prevent errors, so that would be a nice benefit.

**Comment by fraimondo** [comment]
Hi! Any news about this PR?

I want to use this package in Spanish (willing to contribute with translations) but I also found that at least 3 strings are not in any .po file but the English and transifex shows that many are 100% complete. I just went and translated Spanish to 100%, but still can't see these lines:

https://github.com/Bouke/django-two-factor-auth/blob/597edc0925e602ebc3e0ca39437a0f2be6bf4ebd/two_factor/forms.py#L186
https://github.com/Bouke/django-two-factor-auth/blob/597edc0925e602ebc3e0ca39437a0f2be6bf4ebd/two_factor/forms.py#L189
https://github.com/Bouke/django-two-factor-auth/blob/597edc0925e602ebc3e0ca39437a0f2be6bf4ebd/two_factor/forms.py#L192

**Comment by moggers87** [comment]
@fraimondo sorry about that, I was given admin permissions on the Transifex account but had neglected to push new translations. I've pushed the lastest translations from master - let me know if anything is missing.

**Comment by fraimondo** [comment]
@moggers87 Thanks!

Just did all the Spanish ones... waiting for the new release. Any ETA?

**Comment by patroqueeet** [comment]
same for German. `blocktrans` line breaks are not inside `msgids` so stuff does not show translated.
